kratos/python_scripts/response_functions/response_function_interface.py

```diff
@@ -1,16 +1,5 @@
-
-# ==============================================================================
-#  KratosShapeOptimizationApplication
-#
-#  License:         BSD License
-#                   license: ShapeOptimizationApplication/license.txt
-#
-#  Main authors:    Geiser Armin, https://github.com/armingeiser
-#
-# ==============================================================================
-
-class ResponseFunctionBase(object):
-    """The base class for response functions. Each response function
+class ResponseFunctionInterface(object):
```

Comment:
I find it a bit unlucky, that now there are two folders called "response_functions" in the core. One under "python_scripts" and one top-level in the core for the c++ files. Not sure how to avoid this, though.

Reply:
Maybe also adding "interface" or similar to this folder name?

Reply:
Well, if its something like "interface" then for sure there is only this file that fits inside such that we could pull it out anyways. In light of the strict separation between programming languages in both folders, it's maybe not too bad to leave it as is.